src/components/content/rss/rss-render-options.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  documentToHtmlString,
+  Options,
+} from '@contentful/rich-text-html-renderer';
+import { BLOCKS } from '@contentful/rich-text-types';
+import { ContentfulCustomModel } from '../../../types/contentful.types';
+import { Converter } from 'showdown';
+
+enum EmbeddedAssetType {
+  IMAGE = 'image',
+  VIDEO = 'video',
+}
+
+const converter: Converter = new Converter();
+const markdownToHyperText = (markdown: string) => converter.makeHtml(markdown);
```

Comment:
Why does our markdown converter from rich-text not work here? It should do the same.

Reply:
I ran into issues with `parcel` when I tried using the existing HTML AST solution. @dschu-lab might know more about this, as he suggested using a different converter in the first place. I will ask him when he's back.